packages/@aws-cdk-testing/framework-integ/test/aws-cloudwatch/test/integ.dashboard-variables.ts

```diff
@@ -0,0 +1,44 @@
+import { App, Stack, StackProps } from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { Dashboard, Metric, Stats, GraphWidget, SearchDashboardVariable, VariableInputType, VariableType } from 'aws-cdk-lib/aws-cloudwatch';
+
+class DashboardVariablesIntegrationTest extends Stack {
+  constructor(scope: App, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const dashboard = new Dashboard(this, 'Dash');
+
+    const widget = new GraphWidget({
+      title: 'My fancy graph',
+      left: [
+        new Metric({
+          namespace: 'AWS/S3',
+          metricName: 'BucketSizeBytes',
+          label: '[BucketName: ${PROP(\'Dim.BucketName\')}] BucketSizeBytes',
+          statistic: Stats.MAXIMUM,
+          dimensionsMap: { StorageType: 'StandardStorage', BucketName: 'my-bucket' },
+        }),
+      ],
+    });
+
+    // The dashboard variable which changes BucketName property on the dashboard
+    dashboard.addVariable(new SearchDashboardVariable({
```

Comment:
can we integ test ValueDashboardVariable as well please. Can be in same integ test.

Reply:
Added a ValueDashboardVariable into integ test